codegen/generator/nodejs/templates/functions.go

```diff
@@ -1,22 +1,44 @@
 package templates
 
 import (
+	"context"
 	"regexp"
 	"strings"
 	"text/template"
 
+	"dagger.io/dagger"
 	"github.com/iancoleman/strcase"
 
+	"github.com/dagger/dagger/codegen/generator"
 	"github.com/dagger/dagger/codegen/introspection"
 )
 
+var currentSchema *introspection.Schema
+
+func init() {
```

Comment:
We shouldn't use `dagger.Connect()` in an `init()` function

Reply:
Yep, this is an error, it was supposed to be there for my tests. I'm gonna move it into the tests.

Reply:
fixed.